nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/main/java/org/nuxeo/ecm/platform/ec/notification/service/NotificationService.java

```diff
@@ -79,6 +79,13 @@
 
     private static final Log log = LogFactory.getLog(NotificationService.class);
 
+    /**
+     * The key to the config turning on or off autosubscription.
+     *
+     * @since 11.1
+     */
+    public static final String AUTOSUBSCRIBE_CONFIG_KEY = "org.nuxeo.ecm.platform.comment.service.notification.autosubscribe";
```

Comment:
This constant should not be here but where it's used, as it's related to a comments feature, not a notification one.

Reply:
Yes, but not useful if no notification service available so I ended up putting it here. wdyt ?

Reply:
But it's related to comments, so it should go to the comment service. BTW, you assume here https://github.com/nuxeo/nuxeo/pull/3663/files#diff-c765ee349f84c6b796ac98a000e4be25R279 that the Notification Service is always available in the comment context.